app/controllers/classrooms_controller.rb

```diff
@@ -0,0 +1,81 @@
+class ClassroomsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_classroom, only: [:show]
+  before_action :set_classroom, only: [:show, :edit, :update, :destroy, :add_students]
+
+  def index
+    Rails.logger.debug("Current User: #{current_user.inspect}")
```

Comment:
Do you still need to leave this debugger?

Reply:
its fine to remove it it was needed while development right now is not needed anymore